deploy/stacks/container.py

```diff
@@ -445,7 +445,7 @@ def create_task_role(self, envname, resource_prefix, pivot_role_name):
                         'sts:AssumeRole',
                     ],
                     resources=[
-                        f'arn:aws:iam::*:role/{pivot_role_name}',
+                        f'arn:aws:iam::*:role/{pivot_role_name}-*',
```

Comment:
what if pivot_role_name is `dataallPivotRole-cdk` with no regional suffix (feature flag is disabled in `config.json`)? Should this instead be `f'arn:aws:iam::*:role/{pivot_role_name}*'` (with no `-`)

Reply:
Fixed in https://github.com/data-dot-all/dataall/pull/1064/commits/66e65b0b54db09614cd23177e82f93249983db55 Great catch